GUI/dashboard.py

```diff
@@ -1,49 +1,176 @@
 from tkinter import *
 #import <api>
 from PIL import ImageTk, Image
+
```

Comment:
`import zip`

- [ ] import  zip.py as module in this script

- [ ] call zip.GUI(x,y) and put `x`= sheet number (1 or 2) & `y`= path of file

Reply:
It would make sense if we call zip function from `gui.py` rather than reverse

Reply:
Yes, I am doing this only. Check **Line 16** of the updated code in dashboard.py. And then called the main fucntion passing the parametes. See **Line 55** for this.